contents/handbook/small-teams/product-analytics/objectives.mdx

```diff
@@ -1,12 +1,16 @@
-## Q4 2023 Goals
+## Q1 2024 Goals
 
-1. Project "NoteForce Exploration 3000" (Michael, Thomas)
-  - Pair up with Team Replay to complete projects Data Exploration, Notebooks and PostHog 3000.
-  - Release new navigation and insight editor.
-  - Work more HogQL prompts into the interface.
-  - Build metrics into the system, introduce Metric Mondays.
+1. HogQL & Data Exploration (Julian, Marius)
+  - Convert ALL our insights to use HogQL as their base.
```

Comment:
Maybe a few words why that is desirable?

Reply:
+1 - Something about the fact we want to have consistent and reliable queries and that the HogQL base unlocks that?

Reply:
I added more words